### PR TITLE
Cleaned up UI

### PR DESCRIPTION
**Pull request by sphinxc0re**
- fixed welcome screen logo resolution on highDPI screens
- sidebar widget is now taking up 30% of the screen width for better UX
- fixed menu bar entries on macOS
- changed commenting shortcut to Ctrl/⌘ + Enter

### CODE REVIEW

**Comment by sphinxc0re** [comment]
I would like someone to test this on a different OS

File: qml/Panopticon/CommentOverlay.qml [hunk 1]
```diff
@@ -152,7 +152,7 @@ MouseArea {
 				Layout.row: 2
 				Layout.alignment: Qt.AlignRight
 
-				text: "Comment (Shift+Return)"
+				text: "Comment (Ctrl/⌘+Return)"
```
Comment (sphinxc0re) [inline qml/Panopticon/CommentOverlay.qml]: I would like this text to be changed depending on the operating system. Any suggestions?

Comment (m4b) [inline qml/Panopticon/CommentOverlay.qml]: if it was rust could simply do a `cfg` on the os; i dunno how in qml

**Comment by m4b** [comment]
Only thing I'd say is that at some future point I'd like to be able to move the sidebar's width; or have a collapse button even

**Comment by sphinxc0re** [comment]
@m4b Yes, I'm definitely with you on this but for now I think this is okay

File: qml/Panopticon/CommentOverlay.qml (hunk 1 above)
Comment (sphinxc0re) [inline qml/Panopticon/CommentOverlay.qml]: One moment... (pushing changes)

Comment (sphinxc0re) [inline qml/Panopticon/CommentOverlay.qml]: In general it is `⌘` on macOS and `Ctrl` on every other system

**Comment by sphinxc0re** [comment]
Don't know what's wrong with travis. Builds seem to have passed

**Comment by flanfly** [comment]
When you want to know the OS you're running on you can look at the value of `Qt.platform.os`: http://doc.qt.io/qt-5/qml-qtqml-qt.html#platform-prop.

**Comment by sphinxc0re** [comment]
@flanfly I know, I already did that in 6f62330e6ea01ff12259ddc9a2c2a390d7aabf98

**Comment by sphinxc0re** [comment]
All tests are green although the GH UI doesn't show it

**Comment by sphinxc0re** [comment]
I restarted the jobs

File: qml/Panopticon/Window.qml [hunk 2]
```diff
@@ -146,7 +153,7 @@ Ctrl.ApplicationWindow {
 			id: bar
 			anchors.top: parent.top
 			anchors.bottom: parent.bottom
-			width: 250
+			width: parent.width * 0.3
```
Comment (flanfly) [inline qml/Panopticon/Window.qml]: At least cap the width. Maximizing the windows results in a huge sidebar.

Comment (sphinxc0re) [inline qml/Panopticon/Window.qml]: Okay what maximum width are you suggesting?

**Comment by flanfly** [comment]
400px should be enough.

**Comment by flanfly** [comment]
👍

**Pull request merged**